bc_obps/reporting/tests/models/test_report_emission.py

```diff
@@ -3,14 +3,13 @@
 import pytest
 from registration.tests.constants import TIMESTAMP_COMMON_FIELDS
 from reporting.tests.utils.constants import REPORT_DATA_MODELS_COMMON_FIELDS
-from reporting.tests.utils.report_data_bakers import report_emission_baker
 from django.core.exceptions import ValidationError
 
 
 class ReportEmissionModelTest(BaseTestCase):
     @classmethod
     def setUpTestData(cls):
-        cls.test_object = report_emission_baker()
```

Comment:
Facilities are created in the chain of creating this, and I was hitting errors about duplicate names when trying to use recipes below